admin/app/components/solidus_admin/ui/pagination/component.rb

```diff
@@ -1,4 +1,42 @@
 # frozen_string_literal: true
 
 class SolidusAdmin::UI::Pagination::Component < SolidusAdmin::BaseComponent
+  def initialize(page:, path: nil)
+    @page = page
+    @path = path
+  end
+
+  def path_for(page_number)
+    if @path.nil?
+      "#"
```

Comment:
Are we considering `@path` to be `nil` because of the preview? I think that maybe we can isolate that within the preview itself by creating a dummy path resolver there: `->(_page_number) { "#" }`. What do you think?